Fix integration tests should panic if error occurs (#137)
Concordium/concordium-smart-contract-tools: cargo-concordium/CHANGELOG.md

## Unreleased changes

> - Fix integration tests should panic if error occurs.

Reviewer: This message is not very clear. Also the tool does not panic when tests fail now. It just  prints an error and returns a non-zero status code.

> - Fix a bug so that a non-zero status code is now returned by the integration tests if they fail to run successfully.

Reviewer: ```suggestion
- Fix a bug so that a non-zero status code is now returned by `cargo concordium test` if tests fail.
```

Reviewer: Note that also if unit tests failed the status code was 0.
